### PR TITLE
(DO NOT MERGE) Draft for OpenShift Appliance user guide as KCS

### PR DESCRIPTION
**Pull request by skopacz1**
This is just a dummy PR to host my WIP KCS for the OpenShift Appliance user guide on GitHub, so we can discuss feedback and make suggestions directly on the lines. 

I created a new file, docs/draft-user-guide.md, in order to draft the work.

### CODE REVIEW

File: docs/draft-user-guide.md [hunk 1]
```diff
@@ -0,0 +1,641 @@
+The `openshift-appliance` utility enables self-contained OpenShift Container Platform cluster installations, meaning that it does not rely on internet connectivity or external registries. It is a container-based utility that builds a disk image that includes the [Agent-based Installer](https://cloud.redhat.com/blog/meet-the-new-agent-based-openshift-installer-1). The disk image can then be used to install multiple OpenShift Container Platform clusters.
+
+## Downloading the OpenShift-based Appliance builder
+The OpenShift-based Appliance builder is available for download at: [http://registry.redhat.io/assisted/agent-preinstall-image-builder-rhel9](http://registry.redhat.io/assisted/agent-preinstall-image-builder-rhel9)
```
[the remark this answers is not in the document]
Comment (skopacz1) [reply docs/draft-user-guide.md:4]: I'll update this URL here and the other locations where the old URL is mentioned, thanks

**Comment by danielerez** [comment]
Thanks @skopacz1! Looks good to me. I think you can move forward with the KCS.

**Comment by skopacz1** [comment]
Sounds good @danielerez! Do I need to wait until the 4.16 GA to publish the KCS, or should I publish it now?

**Comment by danielerez** [comment]
> Sounds good @danielerez! Do I need to wait until the 4.16 GA to publish the KCS, or should I publish it now?

Yeah, the appliance is tech-preview for now, so it's fine.

**Comment by skopacz1** [comment]
Closing this PR as the KCS is about to be published with no further feedback

**Pull request closed without merging**